mithril-signer/src/runtime/runner.rs

```diff
@@ -217,7 +217,7 @@ impl Runner for SignerRunner {
             self.services.single_signer.get_party_id(),
             protocol_initializer.verification_key().into(),
             verification_key_signature_encoded,
-            operational_certificate_encoded,
+            operational_certificate_encoded.and_then(|o| o.try_into().ok()),
```

Comment:
You should already have a `ProtocolOpCert` here.
Begining at line `182` above, rename the `operational_certificate_encoded` var to `protocol_operational_certificate` and replace the `key_encode_hex` with a `ProtocolOpCert::new(opcert)`.